### PR TITLE
Remove extraneous filtering

### PR DESCRIPTION
**Pull request by rouzbeh**

### CODE REVIEW

File: agcounts/pow2.py [hunk 1]
```diff
@@ -35,15 +35,8 @@ def resample_to_30hz(
...
-            data.transpose(),
-            sfreq=sample_rate,
-            l_freq=None,
-            h_freq=15,
-            verbose=30,
-        )
-        data = filter.resample(filtered, down=sample_rate / 30).transpose()
-        del filtered
+
+        data = filter.resample(data, down=sample_rate / 30, axis=0)
```
Comment (dgerstel) [inline agcounts/pow2.py:39]: Does `filter.resample(.)` apply the low pass filter with the 15 Hz cut-off?

Comment (rouzbeh) [reply agcounts/pow2.py:39]: Yes resample takes care of filtering.

File: tests/test_pow2.py [hunk 2]
```diff
@@ -32,4 +32,4 @@ def test_32hz(input_32hz, output_30hz):
 def test_32hz_interpolate(input_32hz, output_30hz):
     resampled = resample_to_30hz(input_32hz, sample_rate=32, use_mne_filter=True)
     assert np.abs(resampled - output_30hz).max() < 1
-    assert resampled.mean() == pytest.approx(-5.8447899368136745e-05)
+    assert resampled.mean() == pytest.approx(-5.443187660227651e-05)
```
Comment (dgerstel) [inline tests/test_pow2.py:35]: That roughly 7% different result. Is it acceptable?

Comment (rouzbeh) [reply tests/test_pow2.py:35]: We actually get closer to the other way of calculating (the old way)

**Review by dgerstel — APPROVED** [review]

**Pull request merged**